zarf.schema.json

```diff
@@ -382,8 +382,7 @@
       "additionalProperties": false,
       "type": "object",
       "required": [
-        "name",
-        "namespace"
```

Comment:
While namespace is required for charts, since having the namespace set through an imported component is possible namespace isn't truly required in the schema if you have import. There is no way for jsonschema to check values across objects (charts cannot check values on the component object) so we do this check with runtime validation instead, which happens after compose.

Reply:
resolving as this happened in another pr